oxide/types.go

```diff
@@ -4313,6 +4358,30 @@ type NameOrIdSortMode string
 // NameSortMode is sort in increasing order of "name"
 type NameSortMode string
 
+// NetworkAddressIpAddr is the type definition for a NetworkAddressIpAddr.
+//
+// Required fields:
+// - IpAddr
+type NetworkAddressIpAddr struct {
+	IpAddr string `json:"ip_addr,omitempty" yaml:"ip_addr,omitempty"`
+}
+
+// NetworkAddressIeee802 is the type definition for a NetworkAddressIeee802.
+//
+// Required fields:
+// - IEEE802
+type NetworkAddressIeee802 struct {
+	IEEE802 []string `json:"i_e_e_e802,omitempty" yaml:"i_e_e_e802,omitempty"`
```

Comment:
The `i_e_e_e802` JSON field read weird to me but that's how it is in the OpenAPI specification on omicron.